src/animations/FadeIn.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
```

Comment:
I guess this is no longer required so you can just import `ReactElement` as in the `Button` branch.